### ISSUE TITLE
add export button to admin interface to get survey data/images off the pi

### ISSUE THREAD

**Issue by emilyeros** [sharedstreets/curb-wheel#60]
(Dependent on issue 58 and 59)

**Comment by morganherlocker** [comment]
This is in progress with the GeoJSON export issue. My current plan is to add an archive of all images to be included alongside the GeoJSON spans and GeoJSON positions. All three downloads get zipped up into a single file.

**Comment by morganherlocker** [comment]
This is mostly shipped on master, with a couple outstanding data format issues. The data can now be exported from the admin panel here:

![image](https://user-images.githubusercontent.com/814300/82969086-af5f1e00-9f83-11ea-9d76-8935c5bc1031.png)

This issue required adding some additional functions to the graph, which is the main gist of what shipped so far. I also added layout to the admin page that allows for export or resetting of surveys stored on disk. The exported data is a zip compressed archive that currently has a `spans.geojson` file. I'm working out edgecases in the geojson translation code that are causing the geometry to be borked, though the data is valid and otherwise complete.

I also need to add code to include the images archive within the download. This was previously blocked by some data modeling issues that have now been resolved. To move forward, we need to create unique ids upon image upload, which are saved in the graph data structure. Both of the remaining issues are in progress now, and I will close this issue once they have been merged.
